snippets/terraform/README.md

```diff
@@ -0,0 +1,7 @@
+# General Documentation for NGINXaaS terraform resource.
```

Comment:
nit: are you really sure you want to call it `NGINXaaS`?  how am I supposed to pronounce that?

Reply:
😂